files/generate_ansible_command.py

```diff
@@ -234,18 +234,24 @@ def get_hostname(x):
                 playbooks_to_run.add(p)
 
 if update_requirements:
-    commands_to_run.append('sudo /usr/local/bin/update_galaxy.sh')
+    commands_to_run.append({'cmd': 'sudo /usr/local/bin/update_galaxy.sh'})
 
 for p in playbooks_to_run:
     if os.path.exists(p):
-        commands_to_run.append('ansible-playbook -D %s' % p)
+        commands_to_run.append({'cmd': 'ansible-playbook -D %s' % p})
 
 for l in local_playbooks_to_run:
-    commands_to_run.append('sudo /usr/local/bin/ansible_local.py %s' % l)
+    commands_to_run.append({'cmd':
+                            'sudo /usr/local/bin/ansible_local.py %s' % l})
 
 for c in commands_to_run:
     if args.dry_run:
-        print c
+        print c['cmd']
     else:
         syslog.syslog("Running {}".format(c))
-        subprocess.call(c.split(), cwd=args.path)
+        env = os.environ.copy()
```

Comment:
I agree about oversized lines, but it took me a minute to wonder what you where trying to achieve with this loop, seemed too simple to be true :-)